src/crypto/index.ts

```diff
@@ -2844,7 +2883,7 @@ export class Crypto extends TypedEventEmitter<CryptoEvent, CryptoEventHandlerMap
             delete content['io.element.performance_metrics'];
         }
 
-        const encryptedContent = await alg.encryptMessage(room, event.getType(), content);
+        const encryptedContent = await alg.encryptMessage(room, event.getType(), content) as IContent;
```

Comment:
Not sure about this cast. `encryptMessage` returns an object. That is all we know. Wouldn't it be better to update the `encryptMessage` definition instead?

Reply:
yes, but honestly I've not touched this code and I'm sick to the back teeth of trying to land this fix so I just wanted to get something that passed the typechecker.